community/libcap/build

```diff
@@ -11,5 +11,5 @@ mk() {
         "$@"
 }
 
-mk
-mk DESTDIR="$1" install
+make INDENT=true
+make DESTDIR="$1" install
```

Comment:
You can drop the `mk` function in this case.

Reply:
I'd like to see the resulting manifest too. :+1:

Reply:
done, new pr opened.